bus-mapping/src/mock.rs

```diff
@@ -72,26 +73,26 @@ pub struct BlockData {
 }
 
 impl BlockData {
-    /// Create a new block with a single tx that executes the code passed by
-    /// argument.  The trace will be generated automatically with the
-    /// external_tracer from the code.
-    pub fn new_single_tx_trace_code(code: &Bytecode) -> Result<Self, Error> {
+    fn new_single_tx_trace_accounts_gas(
```

Comment:
I would add a minimal documentation for the fn. Any minimal explanation would be fine.